src/protocol/control_layer/broadcast_message/BroadcastMessage.js

```diff
@@ -19,7 +18,7 @@ export default class BroadcastMessage extends ControlMessage {
     }
 
     static create(streamMessage) {
-        return new (ControlMessage.getClass(1, TYPE))(streamMessage)
+        return new (ControlMessage.getClass(ControlMessage.LATEST_VERSION, TYPE))(streamMessage)
     }
 
     static deserialize(messageVersion, broadcastMessageSpecificArgsArray) {
```

Comment:
The point was to remove the individual `deserialize` methods and add the generic impl to the parent (into `ControlMessage.deserialize(msg)`)!